lms/djangoapps/teams/models.py

```diff
@@ -319,25 +319,20 @@ def get_memberships(cls, username=None, course_ids=None, team_ids=None):
         return queryset
 
     @classmethod
-    def user_in_team_for_course(cls, user, course_id, topic_id=None):
+    def user_in_team_for_teamset(cls, user, course_id, topic_id):
         """
-        Checks user membership in two ways:
-        if topic_id is None, checks to see if a user is assigned to any team in the course
-        if topic_id (teamset) is provided, checks to see if a user is assigned to a specific team in the course.
+        Using the provided teamset_id, checks to see if a user is assigned to any team in the teamset.
 
         Args:
             user: the user that we want to query on
             course_id: the course_id of the course we're interested in
-            topic_id: optional the topic_id (teamset) of the course we are interested in
+            teamset_id: the teamset_id (formaerly topic_id) of the course we are interested in
```

Comment:
s/formaerly/formerly

Reply:
Also, this has been changed to teamset_id but it doesn't look like the actual arg name was changed

Reply:
Reverted back to topic_id for now.